### PR TITLE
migrate to dotenv

### PR DESCRIPTION
**Pull request by Eschults**
fixes https://github.com/lewagon/teachers/issues/4

### CODE REVIEW

**Comment by Eschults** [comment]
ping @cveneziani for review before merge 👌 🙏

File: clever_cloud/minimal.rb [hunk 1]
```diff
@@ -33,6 +33,7 @@
   gem 'listen', '~> 3.0.5'
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'dotenv-rails'
```
Comment (cveneziani) [inline clever_cloud/minimal.rb:35]: You forgot to remove `figaro` from this Gemfile.

Comment (Eschults) [reply clever_cloud/minimal.rb:35]: thanks 🙏 anything else?

Comment (cveneziani) [reply clever_cloud/minimal.rb:35]: Nope, perfect! :tada:

**Pull request merged**